build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '7.2.1' apply false
-    id 'com.android.library' version '7.2.1' apply false
+    id 'com.android.application' version '7.3.1' apply false
+    id 'com.android.library' version '7.3.1' apply false
```

Comment:
Oh, I also updated the project from AGP 7.2 to 7.3. It was automated by Android Studio, but probably led to some extra junk in this diff